.github/CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ Please send a Pull Request if you have any additions or changes.
 
 Pull requests will be accepted based on the following:
 
-1. The pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/tree/main/.github).
+1. The pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/tree/main/.github/CONTRIBUTING.md).
```

Comment:
Sorry @KieranRobson - this should be linking to the CoC not contributing. I’d change it myself but I can’t edit the file.

Reply:
Its been a rough morning 😆, this has now been fixed!